src/VisualStudio/CSharp/Impl/EventHookup/EventHookupSessionManager.cs

```diff
@@ -35,6 +43,25 @@ internal void EventHookupFoundInSession(EventHookupSession analyzedSession)
                 caretPoint.HasValue &&
                 analyzedSession.TrackingSpan.GetSpan(CurrentSession.TextView.TextSnapshot).Contains(caretPoint.Value))
             {
+                // tooltips text is: Program_MyEvents;      (Press TAB to insert)
+                // GetEventNameTask() gets back the event name, only needs to add a semicolon after it.
+                var eventText = analyzedSession.GetEventNameTask.Result + ";";
+                if (this._toolTipPresenter == null)
```

Comment:
When would this ever be non-null? That would imply that we already have an existing session active, right? In that case, do we need to call CancelAndDismissExistingSessions() or otherwise we've leaked that other session and event handlers and such?

Reply:
added a debug.Assert